jobscheduler/scheduler-component-shared/pom.xml

```diff
@@ -65,6 +69,15 @@
             <groupId>org.hibernate</groupId>
             <artifactId>hibernate-core</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Yeah these scheduler jobs are all in the scheduler-component-shared and they end up in shared so this lib needs to be there. I would've preferred to not have to put this in shared.